src/ZXing.Net.Maui/.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Should we really push smth vscode related into the repo?

Reply:
It was in original repo

Reply:
of course not, @webwarrior-ws

Reply:
Removed the file